app/models/file_attachment/revision.rb

```diff
@@ -21,6 +21,7 @@ class FileAttachment::Revision < ApplicationRecord
            :ensure_assets,
            :content_type,
            :byte_size,
+           :pdf_page_count,
```

Comment:
I'd suggest we don't prefix this with pdf, although this is the only format we support it on it could well be used for other formats if we could support it. I'm hoping this can help sidestep us away from having to handle file type metadata individually.

I'd also suggest we name this number_of_pages for Whitehall consistency. I suggest this because Whitehalls usage is already in govspeak and it'd be good to use same name in govuk_publishing_components.

Reply:
OK, I wasn't aware there would be other formats with multiple pages.  I figured prefixing it would make it clear what format it's for.  I'll change to `number_of_pages`.

Reply:
There's no plans for other formats to support it - and we may well never add another one. I meant more that there's no need to prefix it for pdf as if we were to ever support it for another format we could just re-use the field.